src/index.html

```diff
@@ -10,7 +10,31 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form 
+      action="https://www.google.com/" 
+      method="get" 
+      data-qa="big">
+      <label>
+        <input
+          class="search_bar1 search_bar"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <div class="space20pxHeight"></div>
```

Comment:
Looks a little weird :)
Don't use empty blocks to create some margins.
Add class name to the `form` and use margin for it.